test/parallel/test-https-simple.js

```diff
@@ -108,6 +97,6 @@ server.listen(0, function() {
   });
 });
 
-process.on('exit', function() {
-  assert.strictEqual(successful, tests);
-});
+function testSucceeded (){
+  server.close();
```

Comment:
Since this is only called from one place, you can just inline the `server.close()` there.